server/routerlicious/packages/routerlicious/src/scribe/index.ts

```diff
@@ -129,13 +138,15 @@ export async function scribeCreate(
 	return new ScribeLambdaFactory(
 		operationsDbManager,
 		documentRepository,
+		checkpointRepository,
 		scribeDeltas,
 		producer,
 		deltaManager,
 		tenantManager,
 		serviceConfiguration,
 		enableWholeSummaryUpload,
 		getDeltasViaAlfred,
+		localCheckpointEnabled,
```

Comment:
I think you can move `localCheckpointEnabled` to `checkpointRepository` to minimize changes outside and hide details there?